legal-api/src/legal_api/__init__.py

```diff
@@ -55,7 +55,7 @@ def create_app(run_mode=os.getenv('FLASK_ENV', 'production')):
     db.init_app(app)
     rsbc_schemas.init_app(app)
     flags.init_app(app)
-    queue.init_app(app)
+    # queue.init_app(app)
```

Comment:
debug?  can we uncomment this?

Reply:
oops!! Yes exactly haha, thank you for catching that